src/Slider/useSlider.tsx

```diff
@@ -221,8 +225,11 @@ export const useSlider = (
   };
 
   const returnCustomDots = (index: number): ReactNode =>
```

Comment:
```
Rewrite to make it more readable. Now, the function not only returns a custom dot, but I think it would be better to rename it to a more suitable name

 const returnCustomDots = (index: number): ReactNode =>{
    if(slideIndex === index){
      return customActiveDot || <ActiveDot activeDotColor={activeDotColor} />
     }
 
     return customDot || (
         <Dot index={index} slideIndex={slideIndex} dotColor={dotColor} />
     );
 }
```